ratis-proto/src/main/proto/Raft.proto

```diff
@@ -296,6 +296,10 @@ message ForwardRequestTypeProto {
 message ReadRequestTypeProto {
 }
 
+message ReadIndexRequestTypeProto {
+
+}
```

Comment:
Let's add a field to ReadRequestTypeProto.
```proto
message ReadRequestTypeProto {
  bool preferNonLinearizable = 1;
}
```

Reply:
done